docs/tour/future.md

```diff
@@ -1,21 +1,189 @@
+to be desired. Dagre is unfortunately unmaintained, and only produces hierarchical
+layouts. Even if it were to be revived, the research papers it's based off of will not
+evolve out of hierarchical layouts. It also doesn't support containers, which is a
+non-starter for software architecture diagrams (D2 has shimmed support for it [2]).
+
+### Plan
+
+1. Integrate with special-purpose layout engines which are "solved". For example, there's
+no improvements to be made to a sequence diagram layout engine. Likewise for a git-commit
+timeline, or radial layouts, among others.
+1. We are developing a new general-purpose layout engine from scratch [3], testing
+exclusively on software architecture diagrams. It can support multiple sub-types in one
+diagram, e.g. part of the diagram is a tree, but not all of it has to be. It handles
+containers and clusters very well, moves labels and icons to avoid collisions, and much
+more.
+
+## Aesthetics
+
+Documentation is only useful if it's consumed, and beautiful diagrams are consumed far
+more than ugly ones. For algorithm-generated diagrams to be a suitable replacement to
+hand-made, it must be at least as aesthetic. To be consistently preferable to hand-made
+ones [4], it must be aesthetic out of the box. Customization should then be graduated
+levels of how much effort you want to expend. Least effort: choose from pre-made themes
+designed by professional designers. More effort: make and edit your own themes. Most
+effort: custom-style everything. Most engineers are satisfied at the first and second
+levels, but D2 must have good support for granular customization.
+
+### How success is measured
+
+- If you're making a diagram for a large audience (e.g. a blog, company all-hands), would
+  you use D2 to make it?
+
+One deciding factor is the time it takes to get to a presentable diagram. Presentable
+doesn't mean full of vibrant colors and shadows and design fluff. It does mean consistent
+colors that increase clarity, on-brand palette of company colors, distinct shapes and
+arrow -- properties that make a professional diagram you'd be proud of presenting.
+
+Another consideration is how customizable the aesthetics are, and how much effort that
+customization takes. I wish the bar here could be that customizing via text can get better
+than the GUI, but I don't believe this to be a possible outcome. If you want a slightly
+different shade of green, are you going to edit the hex code by typing it? Or would you
+rather point and click from a gradient. If you want to change the style of 3 objects
+you're looking at, you can text-search for the names of each of those 3 objects and change
+them individually. But most of us would rather use the superior input device for precision
+targeting -- a mouse.
+
+While getting the customization experience to be GUI-level is merely aspirational, D2 can
+aim to be at least as good as an existing, battle-tested text-based styling language: CSS.
```

Comment:
```suggestion
aim to be at least as good as the existing, battle-tested text-based styling language: CSS.
```

Reply:
sounds weird, i think "an" is the right word here